homeassistant/components/auth/__init__.py

```diff
@@ -598,6 +602,50 @@ async def websocket_delete_refresh_token(
     connection.send_result(msg["id"], {})
 
 
+@websocket_api.websocket_command(
+    {
+        vol.Required("type"): "auth/delete_all_refresh_tokens",
+    }
+)
+@websocket_api.ws_require_user()
+@websocket_api.async_response
+async def websocket_delete_all_refresh_tokens(
+    hass: HomeAssistant, connection: websocket_api.ActiveConnection, msg: dict[str, Any]
+) -> None:
+    """Handle delete all refresh tokens request."""
+    tasks = []
+    current_refresh_token: RefreshToken
+    for token in connection.user.refresh_tokens.values():
+        if token.id == connection.refresh_token_id:
+            # Skip the current refresh token as it has revoke_callback,
+            # which cancels/closes the connection.
+            # It will be removed after sending the result.
+            current_refresh_token = token
+            continue
+        tasks.append(
+            hass.async_create_task(hass.auth.async_remove_refresh_token(token))
+        )
+
+    remove_failed = False
+    if tasks:
+        for result in await asyncio.gather(*tasks, return_exceptions=True):
+            if isinstance(result, Exception):
+                getLogger(__name__).error(
+                    "During refresh token removal, the following error occurred: %s",
+                    result,
+                )
```

Comment:
Since the exception is unexpected we may want a stack trace here.
Either change log level to exception, or pass `exc_info=True` to the logger.